pkg/trace/pb/stats.proto

```diff
@@ -25,13 +25,15 @@ message ClientStatsPayload {
 	string tracerVersion = 6;
 	string runtimeID = 7;
 	uint64 sequence = 8;
+	string agentAggregation = 9;
```

Comment:
Can we please document these two fields? But ideally all of them, including the other structs!